src/index.html

```diff
@@ -2,11 +2,448 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container-logo">
+        <a href="#home">
+          <img src="./images/logo.png" 
+            alt="moyo-logo" 
+            class="header__nav-logo"
+        >
+        </a>
+      </div>
+          
+      <nav class="header__nav">
+        <ul class="header__nav-menu">
+          <li class="header__nav-element">
+            <a 
+              href="#" 
+              class="is-active header__nav-link"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-element">
+            <a 
+              href="#" 
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-element">
+            <a 
+              href="#" 
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-element">
+            <a 
+              href="#" 
+              class="header__nav-link" 
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-element">
+            <a 
+              href="#" 
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-element">
+            <a 
+              href="#" 
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-element">
+            <a 
+              href="#" 
+              class="header__nav-link"
+            >
+            Photo
+            </a>
+          </li>
+          
+          <li class="header__nav-element">
+            <a 
+              href="#" 
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+
+        </ul>
```

Comment:
```suggestion
          </li>
        </ul>
```
Don't leave empty lines between parent and child elements